code/modules/admin/verbs/lua/lua_state.dm

```diff
@@ -97,9 +97,9 @@ GLOBAL_PROTECT(lua_usr)
 
 /datum/lua_state/process(seconds_per_tick)
 	if(timer_enabled)
-		call_function("__Timer_timer_process", seconds_per_tick)
+		call_function_return_first("__Timer_timer_process", seconds_per_tick)
```

Comment:
Not how I would have done it. I would have done
```
var/result = call_function("__Timer_timer_process", seconds_per_tick)
log_result(result, verbose = FALSE)
```

Reply:
I was thinking of doing it this way, I'll do it this way then